src/apis/CurseForge.js

```diff
@@ -0,0 +1,21 @@
+const { APIWrapper } = require('../')
+const fetch = require('node-fetch')
+
+const API_URL = 'https://addons-ecs.forgesvc.net/api/v2'
+
+module.exports = class CurseForge extends APIWrapper {
+  constructor () {
+    super({
+      name: 'curseforge'
+    })
+  }
+
+  searchAddon (gameId, query) {
+    return this.request('/addon/search', { gameId, pageSize: 3, searchFilter: query, sectionId: 4471, sort: 0 })
+  }
+
+  request (endpoint, queryParams = {}) {
+    const qParams = new URLSearchParams(queryParams)
+    return fetch(API_URL + endpoint + `?${qParams.toString()}`).then(res => res.json())
```

Comment:
Use Axios, as per #1096.

Reply:
not required, as per https://github.com/SwitchbladeBot/switchblade/issues/1096#issuecomment-655563952